app/repositories/UIJourneySessionDataRepository.scala

```diff
@@ -105,4 +105,15 @@ class UIJourneySessionDataRepository @Inject()(
       .deleteOne(dataFilter(data))
       .toFuture()
       .map(_ => true)
+
+  private def sessionFilter(sessionID: String): Bson = {
```

Comment:
this method is redundant, not sure if we really need it

Reply:
This was required for the deleteOne(sessionId: String) method

Reply:
yes, but deleteOne you added is redundant